builds/jenkins-build-sls-app/Jenkinsfile

```diff
@@ -6,10 +6,10 @@ import groovy.transform.Field
 
 // To be replaced as @Field def repo_credential_id = "value" for repo_credential_id, repo_base and repo_core
 
-@Field def repo_credential_id
-@Field def repo_base
-@Field def repo_core
-@Field def scm_type
+@Field def repo_credential_id = "jazz_repocreds"
+@Field def repo_base = "slsapp19-gitlab-alb-1957372724.us-east-1.elb.amazonaws.com"
+@Field def repo_core = "slf"
+@Field def scm_type = "gitlab"
```

Comment:
Remove all hard coded values

Reply:
That creeps inside over and over again ;-( Removing once again now

Reply:
Done deal :-)